README.md

```diff
@@ -1,39 +1,80 @@
-# collection_template
-You can build a new repository for an Ansible Collection using this template by following [Creating a repository from a template](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-from-a-template). This README.md contains recommended headings for your collection README.md, with comments describing what each section should contain. Once you have created your collection repository, delete this paragraph and the title above it from your README.md.
 
-# Foo Collection
-<!-- Add CI and code coverage badges here. Samples included below. -->
-[![CI](https://github.com/ansible-collections/REPONAMEHERE/workflows/CI/badge.svg?event=push)](https://github.com/ansible-collections/REPONAMEHERE/actions) [![Codecov](https://img.shields.io/codecov/c/github/ansible-collections/REPONAMEHERE)](https://codecov.io/gh/ansible-collections/REPONAMEHERE)
 
-<!-- Describe the collection and why a user would want to use it. What does the collection do? -->
+# Community Network Collection
+[![Run Status](https://api.shippable.com/projects/5e7c598d1c7dc1000762800c/badge?branch=master)](https://app.shippable.com/github/ansible-collections/community.network/dashboard) [![Codecov](https://img.shields.io/codecov/c/github/ansible-collections/community.network)](https://codecov.io/gh/ansible-collections/community.network)
 
-## Tested with Ansible
+The Community Network collection includes community maintained content to help automate network appliances.
 
-<!-- List the versions of Ansible the collection has been tested with. Must match what is in galaxy.yml. -->
-
-## External requirements
-
-<!-- List any external resources the collection depends on, for example minimum versions of an OS, libraries, or utilities. Do not list other Ansible collections here. -->
 
 ### Supported connections
-<!-- Optional. If your collection supports only specific connection types (such as HTTPAPI, netconf, or others), list them here. -->
+The community network collection supports `network_cli`  and `httpapi` connections.
 
 ## Included content
 
-<!-- Galaxy will eventually list the module docs within the UI, but until that is ready, you may need to either describe your plugins etc here, or point to an external docsite to cover that information. -->
+Click the `Content` button to see the list of content included in this collection.
+
+## Installing this collection
+
+You can install the community network collection with the Ansible Galaxy CLI:
+
+    ansible-galaxy collection install community.network
 
+You can also include it in a `requirements.yml` file and install it with `ansible-galaxy collection install -r requirements.yml`, using the format:
+
+```yaml
+---
+collections:
+  - name: community.network
+    version: 0.1.0
```

Comment:
Is the version required here? If yes, we should better find a way to make sure this is up-to-date when releasing a new version. If not, it might be better to remove this to avoid this being out of date soon...

Reply:
doesn't look like it's required so I'll remove, thanks!

Reply:
How about:
```suggestion
    # If you need a specific version of the collection, you can specify it here:
    # version: ...
```

Reply:
Yup, I like Felix's sugestion/